_posts/2017-04-16-radio-antenna-workgroup-meet.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Radio & Antenna Workgroup Meet
+slug: radio-antenna-workgroup-meet
+text: TODO.
```

Comment:
Maybe just grab the first sentence below?

"A hands on work session to test and configure long-range Ubiquiti antennas."